client/src/components/QA/AnswerModal.jsx

```diff
@@ -95,49 +163,56 @@ class AnswerModal extends React.Component {
                 value={this.state.answer}
                 error={!!this.state.answerError}
                 helperText={this.state.answerError}
-                onChange={e => this.setState({answer: e.target.value})}
+                onChange={this.handleInput}
                 required>
               </TextField>
             </Stack>
             <br></br>
             <Stack spacing={1}>
               <TextField
+                id='Email'
                 label='Email'
                 placeholder="example@atelier.com"
                 fullWidth
                 inputProps={{maxLength: 60}}
                 value={this.state.email}
                 error={!!this.state.emailError}
                 helperText={this.state.emailError}
-                onChange={e => this.setState({email: e.target.value})}
+                onChange={this.handleInput}
```

Comment:
I suggest you give this function a more meaningful name, we know that it's an on change, what does it mean to handle the input? Validate input?

Reply:
Replaced w/ a more meaningful name